icarus/egg-pterodactyl-icarus--dedicated.json

```diff
@@ -1,32 +1,32 @@
 {
-    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "update_url": null,
-        "version": "PTDL_v2"
+        "version": "PTDL_v2",
+        "update_url": null
     },
-    "exported_at": "2024-06-01T00:03:56+00:00",
+    "exported_at": "2025-02-09T15:38:19+00:00",
     "name": "Icarus-Dedicated",
     "author": "bolverblitz@ebg.pw",
     "description": "Icarus is a survival game that with dedicated servers as a public beta",
     "features": [
         "steam_disk_space"
     ],
     "docker_images": {
-        "Wine Staging": "ghcr.io/parkervcp/yolks:wine_latest"
+        "Wine Staging": "ghcr.io\/parkervcp\/yolks:wine_staging"
```

Comment:
This one sas staging and uses staging so you forgot to edit one of the 2?

Reply:
I apparently missed that when I was testing things.

Reply:
I forgot that during testing, sorry

Reply:
@BolverBlitz You tested with latest or the staging image?

Reply:
It works with both right now.

Reply:
Updated the images for both to latest